motion/p5js/motion-triangle/index.js

```diff
@@ -0,0 +1,75 @@
+function Color(color){
+  return "rgb(" + color.r + "," + color.g + "," + color.b + ")";
+}
+colors = {
+  blue:   Color({ r: 0, g: 38, b: 255 }) ,
+  yellow: Color({ r: 240, g: 250, b: 0 }) ,
+  red :   Color({ r: 255, g: 0, b: 0 }),
+};
+
+
+var CANVAS_WIDTH  = 500;
```

Comment:
The canvas width and height are stored in the `width` and `height` variables. If you set the size using `createCanvas` you can access them everywhere in your code. No need to duplicate these with your own variables.